.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,33 @@
+name: Feature request
+description: Suggest an idea for this project
+labels:
+  - 'enhancement'
+body:
+  - type: textarea
+    attributes:
+      label: What would you like to be able to do?
+      description: |
+        Tell us what requirements you need solving, and be sure to mention too if this is part of any "bigger" problem you're trying to solve.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Did you already have any implementation ideas?
+      description: |
+        In case you've already dug into existing options or source code and have ideas, mention them here.
+        Try to keep implementation ideas separate from *requirements* above.
+    validations:
+      required: true
+
+  - type: dropdown
+    attributes:
+      label: Do you want to write this feature yourself?
```

Comment:
Converted the free-form question if people want to help into a streamlined dropdown menu.